test/shlibloadtest.c

```diff
@@ -191,6 +234,8 @@ int setup_tests(void)
         test_type = SSL_FIRST;
     } else if (strcmp(p, "-just_crypto") == 0) {
         test_type = JUST_CRYPTO;
+    } else if (strcmp(p, "-dso_ref") == 0) {
+        test_type = JUST_CRYPTO;
```

Comment:
This doesn't look right. Should that be `test_type = DSO_REFTEST;`?

Reply:
I've incorporated the fix in a larger PR I'm working on. If someone gets to it in the meantime then that's fine too.